docs/docs/Integrations/integrations-instana-traceloop.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Traceloop
+slug: /integrations-instana-traceloop
+description: Instrument Langflow with the Traceloop SDK and export traces and metrics to Instana using OpenTelemetry.
+---
+
+Traceloop SDK is a lightweight instrumentation toolkit designed for LLM (Large Language Model) applications. It enables developers to automatically capture and export traces, metrics, and key observability signals from their LLM-powered workflows.
+
+When combined with Instana, the exported telemetry data from Traceloop provides end-to-end visibility, allowing users to visualize traces, analyze performance bottlenecks, and ensure reliable operation of LLM-driven applications.
+
+This guide demonstrates how to integrate the Instana observability platform with your Langflow application using the Traceloop SDK so you can monitor and analyze LLM performance.
+
+## Prerequisites
+
+- Create a [Traceloop API key](https://app.traceloop.com/settings/api-key)
+- Create an [Instana endpoint and Instana key](https://www.ibm.com/docs/en/instana-observability/1.0.302)
+- [Install and start Langflow](/get-started-installation)
+
+## Configure environment variables
+
+1. In the root directory of your Langflow application, create a `.env` file.
+2. Add your Traceloop and Instana credentials to the `.env` file:
```

Comment:
This should be broken up into steps for each variable. Tell users what this does and why they are adding it, and any configuration issues (like the HTTP/HTTPS setting) to be aware of.